projects/ad9361/src/main.c

```diff
@@ -589,6 +598,11 @@ int main(void)
 		printf("axi_dmac_init rx init error: %"PRIi32"\n", status);
 		return status;
 	}
+	status = axi_dmac_init(&mem_dmac, &mem_dmac_init);
+	if (status < 0) {
+		printf("axi_dmac_init mem init error: %"PRIi32"\n", status);
```

Comment:
nit: `pr_err`

Reply:
I tried to keep the same line as the code existing in main.c. However, these changes will not appear in the final version.